### ISSUE TITLE
Is it possible to perform updates without removing the NPM's plug-in package

### ISSUE THREAD

**Issue by lbyo** [coreybutler/nvm-windows#406]
### My Environment

- [ √] Windows 10

- [ √] My Windows installation is non-English.

### I'm using NVM4W version:

- [ √] 1.1.7

### I have already...

- [√ ] read the README to be aware of npm gotchas & antivirus issues.
- [√ ] reviewed the [wiki](https://github.com/coreybutler/nvm-windows/wiki) to make sure my issue hasn't already been resolved.
- [√ ] verified I'm using an account with administrative privileges.
- [ √] searched the issues (open and closed) to make sure this isn't a duplicate.
- [√ ] made sure this isn't a question about how to use NVM for Windows, since [gitter](https://gitter.im/coreybutler/nvm-windows?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) is used for questions and comments.

### Expected Behavior

- When updating a node with NVM, you should first check whether the NPM needs to be updated before making any updates.
- In other words, keep the relevant plug-ins when updating the NPM.

### Actual Behavior

- In fact, every time you update a node using NVM, you uninstall the NPM and reinstall it, whether it needs to be updated or not.
- Accordingly, after each update, I need to manually download the plug-ins I need.

### Steps to reproduce the problem:
- Open NVM and update node...
- I have seen several similar issues,But I didn't see your exact reply,So how do you plan to solve this problem, or in which version?
- Thank you for your work and bringing us such good software!
